extensions.json

```diff
@@ -1481,6 +1481,11 @@
       "download": "https://github.com/tuxtina/json2yaml/releases/download/v0.2.0/json2yaml-0.2.0.vsix",
       "version": "0.2.0"
     },
+    {
+      "id": "Tyriar.sort-lines",
+      "repository": "https://github.com/Tyriar/vscode-sort-lines",
+      "prepublish": "yarn compile"
```

Comment:
Interesting. According to `vscode:prepublish` in the repository's [package.json](https://github.com/Tyriar/vscode-sort-lines/blob/c22a998b87c8ecdea84db5c8afe7bb8437e1f701/package.json#L11), `npm run compile` will also be run by `vsce` just prior to publishing.

However, since the repository's own [.github/workflows/release.yml](https://github.com/Tyriar/vscode-sort-lines/blob/c22a998b87c8ecdea84db5c8afe7bb8437e1f701/.github/workflows/release.yml#L25-L35) is indeed also set up to run both `yarn compile` followed by `npm run compile`, let's follow the same process here to avoid any subtle differences.